datasm/scripts/publish_to_esgf.py

```diff
@@ -112,7 +113,8 @@ def publish_dataset(args):
             )
             proc.wait()
 
-        con_message("info", f"Return code {str(proc.returncode)} on cmd: {cmd}")      # generated weird error in 3_Publish/slurm_scripts-20211013_185431_334581
+        con_message("info", f"Return code {str(proc.returncode)} on cmd: {cmd}")
+        log_message("info", f"Return code {str(proc.returncode)} on cmd: {cmd}")
```

Comment:
Can we just keep the line for `log_message()`?

Reply:
Yes.  Sterling was real big on flashy/colorful console output inappropriate for batch-mode operation, where all such output gets redirected to log-files anyway.  Post processing of such logs (sorting messages by timestamp) would fail because the leading bytes of message lines are color-codes...